src/fw/board/boards/board_asterix.c

```diff
@@ -154,13 +151,10 @@ void board_early_init(void) {
   nrf_gpio_cfg_output(16);
   nrf_gpio_pin_set(15);
   nrf_gpio_pin_set(16);
-  
-  /* do this now to turn on lfclk */
-  ret_code_t rv; 
-  rv = nrf_sdh_enable_request();
-  PBL_ASSERTN(rv == NRF_SUCCESS);
 
-  PBL_LOG(LOG_LEVEL_ERROR, "SDH enabled");
+  nrf_clock_event_clear(NRF_CLOCK_EVENT_LFCLKSTARTED);
+  nrf_clock_int_enable(NRF_CLOCK_INT_LF_STARTED_MASK);
+  nrf_clock_task_trigger(NRF_CLOCK_TASK_LFCLKSTART);
```

Comment:
either add a TODO here or fix to make sure that lfclk actually comes up; in the field Pebbles with water incursion would fail to have oscillators come up and would produce a sadwatch (0xFE5045xx) rather than doing something unpredictable and wrong, and I could imagine this would be a useful thing to be able to diagnose here

Reply:
done